docs/workflow_generators/residential_hpxml.rst

```diff
@@ -38,43 +41,62 @@ Configuration Example
 Arguments
 ~~~~~~~~~
 
-- ``build_existing_model``: Update the simulation control arguments to the `BuildExistingModel`_ measure. See
-  :ref:`build-existing-model-defaults` for current defaults.
+- ``build_existing_model``: Update the simulation control arguments to the `BuildExistingModel`_ measure.
+  See :ref:`build-existing-model-defaults` for current defaults.
+
+  - ``simulation_control_timestep``: Value must be a divisor of 60.
```

Comment:
Length of simulation timestep in minutes?

Reply:
Yes.